srcpkgs/nxt/files/nxt/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+exec nxt > /dev/null
+
```

Comment:
remove extra blank lines at the end of run files and INSTALL.msg

Reply:
Removed newlines at the end of all files